cvat/apps/annotation/mask_instance.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (C) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+format_spec = {
+    "name": "MASK INSTANCE",
+    "dumpers": [
+        {
+            "display_name": "{name} {format} {version}",
+            "format": "ZIP",
+            "version": "1.0",
+            "handler": "dump"
+        },
+    ],
+    "loaders": [
+    ],
+}
+
+def dump(file_object, annotations):
+    from zipfile import ZipFile
+    import numpy as np
+    import os
+    from pycocotools import mask as maskUtils
+    import matplotlib.image
+    import io
+    from collections import OrderedDict
+
+    # RGB format, (0, 0, 0) used for background
+    def genearte_pascal_colormap(size=256):
```

Comment:
gene**ra**te

I suggest moving this to `pascal_voc.py` and using this function in `mask.py`. Adding of `class masks` export may be useful too.